### PR TITLE
leveraging markdown a bit more

### PR DESCRIPTION
**Pull request by joelhooks**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "presets": ["next/babel"],
-  "plugins": ["istanbul"]
+  "plugins": ["istanbul", "inline-react-svg"]
```
Comment (joelhooks) [inline .babelrc:3]: this lets you import svgs nicely

File: @types/assets/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+declare module '\*.svg' {
```
Comment (joelhooks) [inline @types/assets/index.d.ts:1]: types!

File: components/app/Footer.tsx [hunk 3]
```diff
@@ -7,18 +7,18 @@ function Nav() {
...
           <li>
-            <Link href="/browse">
-              <a>Browse</a>
+            <Link href="/search">
+              <a>Search</a>
             </Link>
           </li>
           <li>
-            <Link href="/courses">
-              <a>Courses</a>
```
Comment (joelhooks) [inline components/app/Footer.tsx:16]: these actually hard redirect in the next config :)

File: components/app/Header.tsx [hunk 4]
```diff
@@ -1,26 +1,26 @@
 import Link from 'next/link'
+import Eggo from '../images/eggo.svg'
```
Comment (joelhooks) [inline components/app/Header.tsx:2]: svg as components ftw

File: pages/browse/index.tsx [hunk 5]
```diff
@@ -1,3 +0,0 @@
-export default function Browse() {
```
Comment (joelhooks) [inline pages/browse/index.tsx:1]: browse might be reconsidered. Learn is the replacement, but killing for now.

File: pages/index.tsx [hunk 6]
```diff
@@ -1,15 +1,14 @@
 import Head from 'next/head'
-import Link from 'next/link'
 
-export default function Home() {
+import Home from '../content/home.mdx'
```
Comment (joelhooks) [inline pages/index.tsx:3]: mdx!

File: pages/learn/react/index.mdx [hunk 7]
```diff
@@ -0,0 +1,4 @@
+# Let's Learn React
```
Comment (joelhooks) [inline pages/learn/react/index.mdx:1]: this is the `/learn/react` page, which is interesting demonstration of our ability to override specific "tags" and topics very easily.

File: pages/stories/index.mdx [hunk 8]
```diff
@@ -0,0 +1,3 @@
+# Humans of egghead
```
Comment (joelhooks) [inline pages/stories/index.mdx:1]: mdx is great for a lot of these types of "curated" pages.

File: pages/talks/[slug].tsx [hunk 9]
```diff
@@ -0,0 +1,151 @@
+import React from 'react'
```
Comment (joelhooks) [inline pages/talks/[slug].tsx:1]: technically just lessons but different.

File: tachyons-to-tailwind-transform.js [hunk 10]
```diff
@@ -0,0 +1,319 @@
+// Installation:
```
Comment (joelhooks) [inline tachyons-to-tailwind-transform.js:1]: this does a codemod on react files that are tachyons and tries to make them tailwind `yarn codemod:tachyons-to-tailwind /path`

File: pages/learn/react/index.mdx (hunk 7 above)
Comment (joelhooks) [reply pages/learn/react/index.mdx:1]: damn, this breaks the build 😭

Any idea why @johnlindquist ?

File: lib/tags.ts [hunk 11]
```diff
@@ -17,15 +17,22 @@ export function getTags() {
...
-        label,
-      },
-    }
-  })(tags)
+  type Tag = {
+    slug: string
+    label: string
+  }
+  return pipe(
+    filter((tag: Tag) => tag.slug != 'react'),
```
Comment (johnlindquist) [inline lib/tags.ts:25]: @joelhooks This fixed the build. The tags.json was generating a path for `learn/react` that was interfering with the mdx file. If we stick to using `tags.json` to generate paths, we could make this filter smarter by checking the `learn` path for existing folders. I'm leaving as a hard-coded `"react"` for now.

Comment (joelhooks) [reply lib/tags.ts:25]: Good catch! Thanks.

**Pull request merged**